lib/puppetserver/ca/action/migrate.rb

```diff
@@ -0,0 +1,95 @@
+require 'puppetserver/ca/utils/cli_parsing'
+require 'puppetserver/ca/utils/file_system'
+require 'puppetserver/ca/utils/http_client'
+
+module Puppetserver
+  module Ca
+    module Action
+      class Migrate
+        include Puppetserver::Ca::Utils
+        PUPPETSERVER_CA_DIR = '/etc/puppetlabs/puppetserver/ca'
+
+        SUMMARY = "Migrate the CA directory out from puppet/ssl/ca to puppetserver/ca"
```

Comment:
"Migrate the CA directory from its current location to `/etc/puppetlabs/puppetserver/ca`"

Since I think this can migrate from wherever it's currently configured, right?

Reply:
Yeah, that's correct.